src/ofxAudioAnalyzerUnit.cpp

```diff
@@ -606,7 +606,7 @@ bool ofxAudioAnalyzerUnit::getIsActive(ofxAAAlgorithm algorithm){
             ofLogWarning()<<"ofxAudioAnalyzerUnit: wrong algorithm to get if is active.";
             break;
     }
-
+  return true;
```

Comment:
this line should be inside the `switch`. `default` should `return false;` or a new `case` should be added

Reply:
adding a return statement to default does not fix the issue... my guess is that gcc is not smart enough to understand that

adding it to the end of the function is still logically consistent though, because ofLogWarning will run, switch case will break, and the function will return.

I'll try to work on a new PR

I do agree that it should be false though